src/Compilers/CSharp/Portable/DocumentationComments/DocumentationCommentIDVisitor.PartVisitor.cs

```diff
@@ -147,7 +147,12 @@ public override object VisitTypeParameter(TypeParameterSymbol symbol, StringBuil
 
                 // Is this a type parameter on a type?
                 Symbol containingSymbol = symbol.ContainingSymbol;
-                if (containingSymbol.Kind == SymbolKind.Method)
+                if (containingSymbol is null)
+                {
+                    // For instance a type parameter in cref (CrefTypeParameterSymbol)
+                    builder.Append('`');
```

Comment:
>builder.Append('`'); [](start = 20, length = 20)

Why do we assume it is a type type parameter? What if it is used in a reference to a generic method instead?

Reply:
If we have a type parameter on a generic method, I think the containing symbol would be the method (or maybe the type, depending where the type parameter comes from).

Reply:
Added a test. For some reason, the type arguments on a method don't get printed out. Not sure if that's by design.

Reply:
> If we have a type parameter on a generic method, I think the containing symbol would be the method (or maybe the type, depending where the type parameter comes from).

What are the grounds for this assumption? It clearly doesn't hold for type type parameters.